$NetBSD$

Silent is gold, ability to debug possible errors as the user wishes
is much more precious (do not filter/redirect stdout/stderr to files
making possible debugging harder).

--- INSTALL.orig	2020-06-22 15:22:27.000000000 +0000
+++ INSTALL
@@ -211,5 +211,5 @@ if $doinstall; then
 	echo "	PLAN9=$PLAN9 export PLAN9"
 	echo '	PATH=$PATH:$PLAN9/bin export PATH'
 fi
-) 2>&1 | tee install.log | $awk -f $PLAN9/dist/isum.awk -v 'copy='install.sum
+)
 
